irods/tests/cfg_irods.yml

```diff
@@ -78,9 +78,13 @@
         - de-create-collection
         - delete-scheduled-rule
         - generateuuid
-        - rm-trash
         - set-uuid
 
+    - name: remove rm-trash if present
+      file:
+        path: /var/lib/irods/msiExecCmd_bin/rm-trash
+        state: absent
```

Comment:
The removal logic needs to be added to the `cfg_irods.yml` playbook, not its unit tests.